scripts/publish.py

```
@@ -0,0 +1,270 @@
+    req = urllib.request.Request(
+        endpoint,
+        data=entry_xml,
+        headers={
+            "Content-Type": "application/atom+xml; charset=utf-8",
+            "X-WSSE": wsse,
+        },
+        method="POST",
+    )
+
+    try:
+        with urllib.request.urlopen(req) as resp:
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)